src/api/resolvers/poapResolvers.js

```diff
@@ -0,0 +1,860 @@
+import { getWeb3Read } from '../web3'
+
+export const defaults = {}
+
+const POAP_ABI = [
```

Comment:
Let's pull this ABI out into it's own file. It's a bit large and hard to find the resolver with such a huge ABI

Reply:
I think we can create a folder called `abis` inside `/api` and add this ABI as well as the token one